diff --git a/src/config/index.ts b/src/config/index.ts
index bf68f37b9..983538b6e 100644
--- a/src/config/index.ts
+++ b/src/config/index.ts
@@ -167,7 +167,7 @@ config = merge(
       // for easier debugging
       debug: {
         startInFatalsLogMode: false, // true setting good for big aws test with nodes joining under stress.
-        startInErrorLogMode: true,
+        startInErrorLogMode: false,
         robustQueryDebug: false,
         fakeNetworkDelay: 0,
         disableSnapshots: true, // do not check in if set to false
diff --git a/src/shardeum/shardeumFlags.ts b/src/shardeum/shardeumFlags.ts
index b48215ac8..be7d0b037 100644
--- a/src/shardeum/shardeumFlags.ts
+++ b/src/shardeum/shardeumFlags.ts
@@ -65,7 +65,7 @@ interface ShardeumFlags {
 export const ShardeumFlags: ShardeumFlags = {
   contractStorageKeySilo: true,
   globalCodeBytes: false,
-  VerboseLogs: false,
+  VerboseLogs: true,
   Virtual0Address: true,
   GlobalNetworkAccount: true,
   FirstNodeRewardCycle: 100,
