diff --git a/src/shardeum/shardeumFlags.ts b/src/shardeum/shardeumFlags.ts
index eae590cc1..bdd49cc25 100644
--- a/src/shardeum/shardeumFlags.ts
+++ b/src/shardeum/shardeumFlags.ts
@@ -109,7 +109,7 @@ export const ShardeumFlags: ShardeumFlags = {
   UseDBForAccounts: true,
   AppliedTxsMaps: false,
   SaveEVMTries: false,
-  ChainID: process.env.CHAIN_ID ? parseInt(process.env.CHAIN_ID) : 8082,
+  ChainID: process.env.CHAIN_ID ? parseInt(process.env.CHAIN_ID) : 8081,
   CheckpointRevertSupport: true,
   UseTXPreCrack: true,
   NewStorageIndex: true,
