diff --git a/src/config/index.ts b/src/config/index.ts
index 80446e7..2193c88 100644
--- a/src/config/index.ts
+++ b/src/config/index.ts
@@ -258,8 +258,8 @@ config = merge(config, {
 
       forwardToLuckyNodes: false, // 1.11.0 we seem to have more issues with this on.  can turn off for local testing
 
-      removeStuckTxsFromQueue: false,
-      removeStuckChallengedTXs: false,
+      removeStuckTxsFromQueue: true,
+      removeStuckChallengedTXs: true,
     },
   },
 })
