diff --git a/src/shardeum/initialNetworkParameters.ts b/src/shardeum/initialNetworkParameters.ts
index d7ebd87ea..68fc6a0c0 100644
--- a/src/shardeum/initialNetworkParameters.ts
+++ b/src/shardeum/initialNetworkParameters.ts
@@ -15,8 +15,8 @@ export const initialNetworkParamters: NetworkParameters = {
   stakeRequiredUsd: oneSHM.mul(new BN(10)),
   maintenanceInterval: ONE_DAY,
   maintenanceFee: 0,
-  minVersion: '1.5.0',
-  activeVersion: '1.5.0',
+  minVersion: '1.4.2',
+  activeVersion: '1.4.1',
   latestVersion: '1.5.1',
   stabilityScaleMul: 1000,
   stabilityScaleDiv: 1000,
