diff --git a/set-stakeAdd-globalAcc.patch b/set-stakeAdd-globalAcc.patch
new file mode 100644
index 0000000..e69de29
diff --git a/src/config/index.ts b/src/config/index.ts
index 4ee3a2d..d8a3b33 100644
--- a/src/config/index.ts
+++ b/src/config/index.ts
@@ -8,11 +8,17 @@ const overwriteMerge = (target: any[], source: any[]): any[] => source // eslint
 let config: {
   storage?: any // eslint-disable-line @typescript-eslint/no-explicit-any
   server: {
+    globalAccount: string
     p2p?: { cycleDuration: number }
     baseDir: string
     mode?: 'debug' | 'release'
   }
-} = { server: { baseDir: './' } }
+} = {
+  server: {
+    globalAccount: '1' + '0'.repeat(63),
+    baseDir: './',
+  },
+}
 
 if (fs.existsSync(path.join(process.cwd(), 'config.json'))) {
   const fileConfig = JSON.parse(fs.readFileSync(path.join(process.cwd(), 'config.json')).toString())
diff --git a/src/setup/sync.ts b/src/setup/sync.ts
index 88696b8..c032bdd 100644
--- a/src/setup/sync.ts
+++ b/src/setup/sync.ts
@@ -21,7 +21,7 @@ function isDebugMode(): boolean {
 
 const oneEth = new BN(10).pow(new BN(18))
 
-export const networkAccount = '0'.repeat(64)
+export const networkAccount = config.server.globalAccount
 
 //In debug mode the default value is 100 SHM.  This is needed for certain load test operations
 const defaultBalance = isDebugMode() ? oneEth.mul(new BN(100)) : new BN(0)
diff --git a/src/shardeum/shardeumConstants.ts b/src/shardeum/shardeumConstants.ts
index 4799b29..5450318 100644
--- a/src/shardeum/shardeumConstants.ts
+++ b/src/shardeum/shardeumConstants.ts
@@ -1,6 +1,7 @@
+import config from '../config'
 import { BN } from 'ethereumjs-util'
 
-export const networkAccount = '0'.repeat(64) //address
+export const networkAccount = config.server.globalAccount
 
 // HELPFUL TIME CONSTANTS IN MILLISECONDS
 export const ONE_SECOND = 1000
diff --git a/src/shardeum/shardeumFlags.ts b/src/shardeum/shardeumFlags.ts
index fcc3125..4d83076 100644
--- a/src/shardeum/shardeumFlags.ts
+++ b/src/shardeum/shardeumFlags.ts
@@ -123,7 +123,7 @@ export const ShardeumFlags: ShardeumFlags = {
   // '1000000000000000000' // $1 * 10 ^ 18
   constantTxFeeUsd: '10000000000000000', // $0.01 * 10 ^ 18
   devPublicKey: 'fe73383a971bd52be51c660cc9758c13763a7d804ee2062b6f7426698148240c', //'fe73383a971bd52be51c660cc9758c13763a7d804ee2062b6f7426698148240c', //'774491f80f47fedb119bb861601490f42bc3ea3b57fc63906c0d08e6d777a592',
-  stakeTargetAddress: '0x0000000000000000000000000000000000000001',
+  stakeTargetAddress: '0x0000000000000000000000000000000000010000',
   certCycleDuration: 10,
   cacheMaxCycleAge: 5,
   cacheMaxItemPerTopic: 4500,
