diff --git a/src/config/index.ts b/src/config/index.ts
index a587472..8892fe8 100644
--- a/src/config/index.ts
+++ b/src/config/index.ts
@@ -320,6 +320,7 @@ config = merge(
         countEndpointStart: -1,
         hashedDevAuth: '',
         devPublicKeys: {
+          '774491f80f47fedb119bb861601490f42bc3ea3b57fc63906c0d08e6d777a592': DevSecurityLevel.High,
           // '': DevSecurityLevel.Unauthorized,
           // These are production keys.  Use 'git apply use_test_key.patch' for unsafe local test keys
           // Never merge a commit with changes to these lines without approval.
