diff --git a/src/config/index.ts b/src/config/index.ts
index 40e78db93..69b4df5e5 100644
--- a/src/config/index.ts
+++ b/src/config/index.ts
@@ -215,9 +215,9 @@ config = merge(
         disableSnapshots: true, // do not check in if set to false
         countEndpointStart: -1,
         hashedDevAuth: '',
-        devPublicKey: '5aab34ddc09a479a835d26d5791dd8f54360cd09c58cd70cd7d818c1af9f8dad', //for production
+        //devPublicKey: '5aab34ddc09a479a835d26d5791dd8f54360cd09c58cd70cd7d818c1af9f8dad', //for production
         //devPublicKey: 'fe73383a971bd52be51c660cc9758c13763a7d804ee2062b6f7426698148240c', //old production
-        //devPublicKey: '774491f80f47fedb119bb861601490f42bc3ea3b57fc63906c0d08e6d777a592', //for local test
+        devPublicKey: '774491f80f47fedb119bb861601490f42bc3ea3b57fc63906c0d08e6d777a592', //for local test
         checkAddressFormat: false,
       },
     },
